dylint_lints/test_validate_gts_docs.py

```
@@ -0,0 +1,493 @@
+        """Hyphen in segment should be detected."""
+        content = """# Types
+
+Type: `gts.my-vendor.core.modkit.plugin.v1~`
+"""
+        with tempfile.NamedTemporaryFile(mode='w', suffix='.md', delete=False) as f:
+            f.write(content)
+            f.flush()
+            path = Path(f.name)
+        
+        try:
+            errors = scan_file(path)
```

Ruff (0.14.11)

Local variable `errors` is assigned to but never used

Remove assignment to unused variable `errors`

(F841)